src/middlewared/middlewared/api/v25_04_0/virt.py

```diff
@@ -178,6 +178,13 @@ class VirtInstanceEntry(BaseModel):
     raw: dict
 
 
+# Lets require at least 32MiB of reserved memory
+# This value is somewhat arbitrary but hard to think lower value would have to be used
+# (would most likely be a typo).
+# Running container with very low memory will probably cause it to be killed by the cgroup OOM
+MemoryType: TypeAlias = Annotated[int, Field(strict=True, ge=33554432)]
```

Comment:
You can use `conint` as well btw

Reply:
And will be deprecated, if I am reading correctly